lib/Yam/Agama/Pom/AgamaUpAndRunningPage.pm

```diff
@@ -22,6 +22,7 @@ sub new {
               agama-unsupported-autoyast-elements
               agama-installing
               agama-sle-overview
+              agama-sle-headless
```

Comment:
We should delete these needles https://openqa.suse.de/tests/17474023#step/boot_agama/8 and check that Agama has booted in the serial for qemu in the same way that is done in remote workers, otherwise having this, if there is a bug that the installation hat expect a desktop end up in a multi-user, we cannot detect this, we would totally miss it.

Reply:
removed and use `wait_serial('Connect to the Agama installer using these URLs:'`